src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallAppsTask.groovy

```diff
@@ -244,7 +248,7 @@ class InstallAppsTask extends AbstractServerTask {
         }
         LooseWarApplication looseWar = new LooseWarApplication(task, config)
         looseWar.addSourceDir()
-        looseWar.addOutputDir(looseWar.getDocumentRoot() , task, "/WEB-INF/classes/");
+        looseWar.addOutputDir(looseWar.getDocumentRoot() , task.classpath.getFiles().toArray()[0], "/WEB-INF/classes/");
```

Comment:
Can you explain this change? Maybe add a comment to the code explaining that second parameter.

Reply:
This was just taken from the refactored `LooseApplication` class. Instead of passing in a task and then resolving the file, I just passed in the file directly. Not sure what exactly that parameter is since I didn't write the original code.